src/app/_components/PillButton/component.js

```diff
@@ -1,3 +1,5 @@
+'use client';
+
 import PropTypes from 'prop-types';
 import { cn } from '@/app/utils/cn';
 import { Paragraph } from '../Typography';
```

Comment:
🟢 change typography import to @component alias

Reply:
thanks, done